mifospay/src/main/java/org/mifos/mobilewallet/mifospay/SplashScreen.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.mobilewallet.mifospay;
+
+import android.content.Intent;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+
+import org.mifos.mobilewallet.mifospay.auth.ui.LoginActivity;
+
+public class SplashScreen extends AppCompatActivity {
+    public static final int splash_screen_timer = 2000;
```

Comment:
Use the camel case naming style please, or as it is a final variable you can write all the letters in the capital.